schemas/blackmarket-README.md

```diff
@@ -0,0 +1,56 @@
+# EDDN BlackMarket Schema
+
+## Introduction
+
+**This schema is deprecated.  The same data has been made available via the
+`prohibited` array in the [commodity schema](./commodity-v3.0.json) since 
+September 2017.**
+
+What follows below is only for historical curiosity.
+
+---
+
+Here we document how to take data from an ED `BlackMArket` Journal Event and
```

Comment:
`BlackMArket` should be `BlackMarket`. ;-)

Reply:
There is no `Blackmarket` journal event. Per line 20 below, "the primary data source for this schema is the ED Journal event `MarketSell`."

Reply:
That'll be me failing to replace copy/pasted parts, will fix later.